frontEnd/tickets-app/src/components/Film/Film.js

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+import "./Film.scss";
+import { Link } from "react-router-dom";
+
+const Film = ({ img, dateEnd, dateStart, name, id }) => {
+  const [start] = useState(new Date(dateStart));
+  const [end] = useState(new Date(dateEnd));
+  return (
+    <div className="wrapper">
+      <Link to={"../movie/" + id}>
+        <img src={img} alt={img} />
+        <br />
+        <span>{name}</span>
+        <br />
+        <span>{`C ${start.getDay()}.${start.getMonth()}.${start.getFullYear()} по ${end.getDay()}.${end.getMonth()}.${start.getFullYear()}`}</span>
```

Comment:
maybe it's better to write a function which converts a date to string:

```
(date) =>  `${date.getDay()}.${date.getMonth()}.${date.getFullYear()}`
```

and you've confused the last date with a start one